### ISSUE TITLE
Better diagnoses on MySQL Installation issues

### ISSUE THREAD

**Issue by aronchick** [kubeflow/pipelines#1354]
Repro:
- Did install of standard Kubeflow using kfctl (non-standard namespace)

Expected:
- Pipeline schedules

Actual:
- Nothing happens (and no K8s logs)

@jlewi

**Comment by Ark-kun** [comment]
Can you provide any additional information?

Can you reach the cluster?
Do you see the UX?
Can you submit pre-packaged pipelines?

**Comment by aronchick** [comment]
The issue is that it is VERY hard to diagnose what went wrong, so it's tough to provide any of these.

Yes, we reached the cluster.
Yes, we saw the UX, but it only rendered the outside frame (the interior of the frame spun forever).
We could not see (nor could we submit) pre-packaged pipelines.

This was a miracle that we were able to debug at all.

The issue was that MySQL did not install correctly, because a standard on many systems is to provision a disk with lost+found in the directory. Unfortunately, the MySQL package for both Pipelines and Katib do not use the "--ignore-data-dir=lost+found" flag (https://www.chriscalender.com/ignoring-the-lostfound-directory-in-your-datadir/), nor does it create its own data directory (e.g. /data-dir) which it can ensure is empty, so it became corrupted when starting.

It'd be much nicer to 

1. diagnose that mysql did not start correctly 
2. provide a way to re-run just the MySQL installation
3. use a data-dir (instead of root) for MySQL installations
4. use the --ignore-data-dir=lost+found flag
5. have docs around central/verbose debugging of setup?

Thank you!
